plugins/main/bridge/bridge.go

```diff
@@ -517,24 +549,6 @@ func cmdAdd(args *skel.CmdArgs) error {
 			return err
 		}
 
-		// check bridge port state
-		retries := []int{0, 50, 500, 1000, 1000}
-		for idx, sleep := range retries {
-			time.Sleep(time.Duration(sleep) * time.Millisecond)
-
-			hostVeth, err := netlink.LinkByName(hostInterface.Name)
```

Comment:
Deleting this line is no bueno :-)
You're no longer refreshing the veth state.

Reply:
It was moved elsewhere.